md/wildfly-bundle.md

```diff
@@ -1,41 +1,41 @@
-# JBoss bundle
+# Wildfly bundle
 
-You will find here steps needed to install and configure a JBoss bundle.
+You will find here steps needed to install and configure a Wildfly bundle.
 
-The JBoss bundle is a zip archive that contains the Red Hat JBoss JEE application server packaged with Bonita BPM Portal and [Bonita configuration folder](BonitaBPM_platform_setup.md#platform_setup_tool).  
-The JBoss bundle is a regular zip archive based on the JBoss zip distribution.
+The Wildfly bundle is a zip archive that contains the Red Hat Wildfly JEE application server packaged with Bonita BPM Portal and [Bonita configuration folder](BonitaBPM_platform_setup.md#platform_setup_tool).  
```

Comment:
instead of saying "and [Bonita configuration folder]", I would rather be more generic: "ready to use with Boita BPM", because there is not only the setup tool shipped with the bundle.